src/js/application.js

```diff
@@ -413,3 +396,18 @@ export class Application {
         this.checkResize(true);
     }
 }
+
+Application.trackClicks = undefined;

```

Comment:
Why not add it as a static member?

```js
class Application {
  static trackClicks = undefined;

}
```

Reply:
Going to do that, I'm also going to do this in the variant-class branch

Reply:
Alright!